examples/plot_wlasso.py

```diff
@@ -10,90 +10,96 @@
 
 # Authors: Quentin Bertrand <quentin.bertrand@inria.fr>
 #          Quentin Klopfenstein <quentin.klopfenstein@u-bourgogne.fr>
-#
+#          Kenan Sehic
 # License: BSD (3-clause)
 
-import time
-
 import numpy as np
 
+from celer import Lasso, LassoCV
+
 from sklearn.datasets import make_regression
 from sklearn.model_selection import train_test_split
-
-from celer import Lasso
+from sklearn.metrics import mean_squared_error
 
 from sparse_ho.models import wLasso
 from sparse_ho.criterion import CV
 from sparse_ho.implicit_forward import ImplicitForward
 from sparse_ho.utils import Monitor
 from sparse_ho.ho import grad_search
-from sparse_ho.datasets import get_data
-
-print(__doc__)
 
-dataset = 'rcv1_train'
-# dataset = 'simu'
-
-if dataset != 'simu':
-    X_train, X_val, X_test, y_train, y_val, y_test = get_data(dataset)
-    X_train = X_train[:, :1000]
-    X_test = X_test[:, :1000]
-else:
-    X, y = make_regression(n_samples=100, n_features=100, noise=1)
-    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.33)
-    X_train, X_val, y_train, y_val = train_test_split(
-        X_train, y_train, test_size=0.5)
+##############################################################################
+# Dataset creation
+##############################################################################
+X, y = make_regression(n_samples=500, n_features=600, noise=0, random_state=2)
```

Comment:
add noise yourself now
noise = np.random.randn(len(y))
y += target_SNR * norm(y) * noise / norm(noise)

Reply:
otherwise the pb is too easy

Reply:
> add noise yourself now
> noise = np.random.randn(len(y))
> y += target_SNR * norm(y) * noise / norm(noise)

why adding ourselves and not using directly the make_regression function?

Reply:
my code above is wrong, it should be divided by target_SNR